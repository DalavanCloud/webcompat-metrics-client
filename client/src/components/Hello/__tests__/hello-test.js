import React from "react"
import renderer from "react-test-renderer"

import Hello from "../"

it("renders Hello default correctly", () => {
  const tree = renderer.create(<Hello />).toJSON()
  expect(tree).toMatchSnapshot()
})
