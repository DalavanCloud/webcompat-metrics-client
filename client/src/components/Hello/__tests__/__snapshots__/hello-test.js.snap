// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Hello default correctly 1`] = `
<div>
  Hello
</div>
`;
